socketmode/socket_mode_managed_conn.go

```diff
@@ -219,6 +219,7 @@ func (smc *Client) connect(ctx context.Context, connectionCount int, additionalP
 				return nil, nil, err
 			}
 		case *slack.RateLimitedError:
+			smc.Debugf("socket mode open and dial: %s", err.(*slack.RateLimitedError).Error())
```

Comment:
How about using `actual` instead of `err.(*slack.RateLimitedError)`?

```suggestion
			smc.Debugf("socket mode open and dial: %s", actual.Error())
```